src/entries/Entry.jsx

```diff
@@ -68,27 +68,29 @@ function Entry({entry, expanded, onExpand}) {
         <Accordion expanded={expanded} onChange={handleChange} style={style} ref={ref}>
             <AccordionSummary expandIcon={<ExpandMoreIcon/>}>
                 <BeltStripe value={entry.belt}/>
-                <div style={{marginRight: 8, width: '50%', flexShrink: 0, flexDirection: 'column'}}>
+                <div style={{margin: '12px 0px 0px 8px', width: '55%', flexShrink: 0, flexDirection: 'column'}}>
                     <FieldValue
-                        name='Make / Model'
+                        // name='Make / Model'
```

Comment:
I don't like to check in commented out code. IMO we should make a prop on FieldValue to hide the label. and then set that to true here. Either that or just don't use FieldValue, but we do get some styling from it I think.

Reply:
Should I delete the commented lines, or wait for the updated FieldValue?